Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/ui/onboarding/CovidCertificateOnboardingViewModel.kt

```diff
@@ -26,7 +29,12 @@ class CovidCertificateOnboardingViewModel @AssistedInject constructor(
         covidCertificateSettings.isOnboarded.update { true }
         val event = if (dccQrCode != null) {
             try {
-                val containerId = dccQrCodeHandler.handleQrCode(dccQrCode)
+                val containerId = dccQrCodeHandler.handleQrCode(
```

Comment:
what about injecting the AppConfig in `DccQrCodeHandler` itself, so you don't need to inject the AppConfigProvider everywhere you need to call the Handler

Reply:
I would rather do that in the `BlocklistValidator` itself

Reply:
it's not used there anymore